pkg/api/helpers.go

```diff
@@ -824,6 +824,47 @@ func (api *API) UpdateQuota(conf QuotaConfig) error {
 	return err
 }
 
+// QuotaBucketConfig quota request
+type QuotaBucketConfig struct {
+	UID        string `url:"uid,ifStringIsNotEmpty"`         // The user to specify a quota
+	MaxObjects string `url:"max-objects,ifStringIsNotEmpty"` // The max-objects setting allows you to specify the maximum number of objects. A negative value disables this setting.
+	MaxSizeKB  string `url:"max-size-kb,ifStringIsNotEmpty"` // The max-size-kb option allows you to specify a quota for the maximum number of bytes. A negative value disables this setting
+	Enabled    string `url:"enabled,ifStringIsNotEmpty"`     // The enabled option enables the quotas
+	QuotaType  string `url:"quota-type,ifStringIsNotEmpty"`  // The quota-type option sets the scope for the quota. The options is bucket .
+	Bucket     string `url:"bucket,ifStringIsNotEmpty"`      // The bucket option is bucket-name
+}
+
+// UpdateQuota update  bucket quota
+//
+// !! caps:	users=write !!
+//
+//@UID
+//@Quota [bucket]
+//
+func (api *API) UpdateBucketQuota(conf QuotaBucketConfig) error {
```

Comment:
UpdateSpecialBucketQuota()

Reply:
```suggestion
func (api *API) UpdateBucketQuota(conf QuotaBucketConfig) error {
```
use method UpdateSpecifiedBucketQuota()